docs/content/users/install/ddev-installation.md

```diff
@@ -210,22 +226,14 @@ Installing and upgrading DDEV are nearly the same thing, because you're upgradin
 
-    !!!warning "Projects go in `/home`, not on the Windows filesystem"
-        Make sure you put your projects in the Linux filesystem (e.g. `/home/<your_username>`), **not** in the Windows filesystem (`/mnt/c`), because you’ll get vastly superior performance on the Linux filesystem. You will be very unhappy if you put your project in `/mnt/c`.
-
-    !!!note "WSL2 hostname management"
-        DDEV attempts to manage custom hostnames via the Windows-side hosts file (usually `C:\Windows\system32\drivers\etc\hosts`) rather than the hosts file inside WSL2. It can only do this if DDEV (with sudo.exe) is installed and in the `$PATH` on the Windows side. (DDEV inside WSL2 uses `ddev.exe` on the Windows side as a proxy to update the Windows hosts file.) If `ddev.exe --version` gives you a reasonable result you're all set up. Otherwise, install DDEV on Windows using `choco install -y ddev` or by downloading and running the Windows installer. In general, the `ddev.exe` on the Windows side does not have to be the exact same version as the `ddev` used in WSL2.
-
     !!!note "Path to certificates"
         If you get the prompt `Installing to the system store is not yet supported on this Linux`, you may just need to add `/usr/sbin` to the `$PATH` so that `/usr/sbin/update-ca-certificates` can be found.
 
-=== "Traditional Windows"
-
-    ## Traditional Windows
+    ### Traditional Windows
 
-    DDEV works fine on the Windows side, but it’s slower than WSL2 by default. Enable either [Mutagen](performance/#system-requirements) or [NFS](performance/#nfs) for the best performance.
+    It is possible to install DDEV in traditional Windows, and it works fine, but it’s massively than WSL2. Enable either [Mutagen](performance/#system-requirements) for the best performance if you have to use traditional Windows, but the best alternative is to install on WSL2.
 
-    * If you use [Chocolatey](https://chocolatey.org/) (recommended), you can run `choco install ddev git` from an administrative shell. Upgrades are just `ddev poweroff && choco upgrade ddev`.
+    * If you use [Chocolatey](https://chocolatey.org/) (recommended), you can run `choco install ddev docker-desktop git` from an administrative shell. Upgrades are just `ddev poweroff && choco upgrade ddev`.
     * A Windows installer is provided in each [DDEV release](https://github.com/drud/ddev/releases) (`ddev_windows_installer.<version>.exe`). Run that and it will do the full installation for you. Open a new Git Bash or PowerShell or cmd window and start using DDEV.
     * Most people interact with DDEV on Windows using Git Bash, part of the [Windows Git suite](https://git-scm.com/download/win). Although DDEV does work with cmd and PowerShell, it's more at home in Bash. You can install it with Chocolatey using `choco install -y git`.
     * For performance, many users enable Mutagen, `ddev config global --mutagen-enabled` (global) or `ddev config --mutagen-enabled` just for one project.
```

Comment:
Omit this point since it’s also not an installation option and we started off this section with this exact information and a link for more detail and examples.

```suggestion
```